src/mcpil.py

```diff
@@ -106,6 +106,9 @@
 proxy_thread: threading.Thread = None
 proxy = Proxy()
 
+# Set icon in taskbar
+window.iconphoto(True, PhotoImage(file="/usr/share/pixmap/mcpil.png"));
```

Comment:
Maybe I’m completely wrong but shouldn’t this have a tilde like so?
`window.iconphoto(True, PhotoImage(file="~/usr/share/pixmap/mcpil.png"));`

Reply:
Tilde means home dir, /usr/share is a non home dir

Reply:
Oop I thought usr was part of home.